docs/guides/postinstall.md

```diff
@@ -184,6 +185,59 @@ S3 suspend has been broken since macOS Sonoma, it has never been fixed, but this
 
 5. If you are facing issues with Wi-Fi on resume, uncomment the lines having `brcmfmac` and `brcmfmac_wcc` in the above file.
 
-!!! note
+> !!! note
```

Comment:
What is the need of > here?

Reply:
Also, for the whole code block, you need to start writing each line after 5 spaces to make it a part of the point. For reference, see the code block of the arch bit in the same guide.